Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 members = [
     "contracts",
+    "e2e",
```

Comment:
I guess we will set this test up in travis too. It's fine if that's another PR.

Reply:
I added it to this PR